### DISCUSSION TITLE
make "permit skip" a parameter per goal instead of global

### DISCUSSION THREAD

**Discussion by ke352802081770314** [iSoron/uhabits#973] [Feature Requests]
Currently, the ability to "skip a day" is a global setting.
I can imagine that it may be useful to be able to permit skipping for some goals but not for others. For example for ones that are dependent on external circumstances, whereas others only depend on willpower/ motivation/ determination, and a "permission to skip" may decrease the commitment.
Therefore it would be better if the ability to skip could be configured per goal, so the user can enable it only for those goals where this flexibility is required/desired.
I am well aware that just the technical ability to enter a skip does not make a statement whether I should make use of it for a particular goal, and the track record will expose having done so, but in the end this app provides some help in doing the right thing, so the better the guidance the more likely there will be a good result.

**Comment by t214c** [comment]
I'd like to extend this request to having all such options be per habit.

- sticky notification or not
- skippable or not
- can have missing data or not

**Comment by baincd** [comment]
I think this would be useful as well.   For example, if I have a habit that is only for working days, I want to be able to mark the habit as skipped on non-working days so it doesn't impact the habit streak or the habit score.

However, I prefer not to enable the global "skip" option, as I don't to be tempted to mark other daily habits as "skipped" when I really just didn't not do them.
